code/src/components/TodoContainer.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components'
+import { AddItem } from 'components/AddItem'
+import { ListItem } from 'components/ListItem'
+import { SettingsContainer } from 'components/SettingsContainer'
+
+const Container = styled.main`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: DodgerBlue;
+  min-height: 100vh;
+`
+
+export const TodoContainer = () => {
+  return(
+    <>
+      <SettingsContainer />
+      <Container>
+        <AddItem />
+        <ListItem />
+      </Container>
+    </>
+  )
```

Comment:
this could be written as an implicitly return:
```
export const TodoContainer = () => (
    <>
      <SettingsContainer />
      <Container>
        <AddItem />
        <ListItem />
      </Container>
    </>
  )
```